test/bdd/pkg/v1/issuer/issuer_steps.go

```diff
@@ -83,8 +83,8 @@ func (e *Steps) createIssuerProfile(user, organizationName string) error { //nol
 			Contexts:         nil,
 			DidMethod:        "orb",
 			Format:           "ldp_vc",
-			KeyType:          "ED25519",
-			SigningAlgorithm: "Ed25519Signature2018",
+			KeyType:          "ECDSAP256DER",
```

Comment:
@aholovko @sudeshrshetty Is there a list of supported key types ? I don't see it in OpenAPI spec (ideally we should have an enum).

Reply:
I can see that we have validation for supported key types depending on concrete key manager (https://github.com/trustbloc/vcs/blob/main/pkg/restapi/v1/issuer/controller.go#L324), so not sure if we can just enumerate supported key types in spec. cc @vkubiv ^

Reply:
yes they depend on kms, also they depend on signatureAlgorithm